src/js/components/SideTabs.js

```diff
@@ -26,9 +26,17 @@ class SideTabs extends React.Component {
     let {selectedTab, tabs} = this.props;
 
     return tabs.map((tab, index) => {
-      let {title, selectValue} = tab;
+      let {title, selectValue, definition} = tab;
+
+      // Check if at least one field has errors
+      let hasErrors = definition && definition.reduce(function (lastErrors, def) {
+        return lastErrors || !!def.showError;
```

Comment:
We typically try to avoid abbreviations, could we rename `def` to `field` or `fieldDefinition`?